src/esphomelib/light/neo_pixel_bus_light_output.h

```diff
@@ -97,6 +99,9 @@ class NeoPixelBusLightOutputBase : public Component, public AddressableLight {
   PowerSupplyComponent *power_supply_{nullptr};
   bool has_requested_high_power_{false};
 #endif
+
+private:
```

Comment:
```suggestion
 protected:
```

a) indent with 1 space as google guidelines say
b) use protected (long story short, I prefer protected for various reasons concerning extensibility)

Reply:
I didn't realize that there is this "suggested change" feature now. Will use it next time...

Reply:
No no, suggested change is only there to make it easier _for you_. I don't care if you use it or not :)

Reply:
Sure! But I think I like it...